Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM rails:4.2.6
+
+RUN echo "deb http://us.archive.ubuntu.com/ubuntu precise main universe" | tee -a /etc/apt/sources.list
+
+RUN mkdir /usr/local/node && curl -L https://nodejs.org/dist/v4.4.3/node-v4.4.3-linux-x64.tar.xz | tar Jx -C /usr/local/node --strip-components=1
+RUN ln -s ../node/bin/node /usr/local/bin/
+RUN ln -s ../node/bin/npm /usr/local/bin/
+
+# For running tests we need phantomjs
+RUN mkdir /usr/local/phantomjs && curl -L https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2 | tar -xj -C /usr/local/phantomjs --strip-components=1
+RUN ln -s ../phantomjs/bin/phantomjs /usr/local/bin/
```

Comment:
It would be good to do a `sha256sum` check here to ensure that the binary is downloaded correctly: http://phantomjs.org/download.html#checksums